src/views/Home.jsx

```diff
@@ -1,15 +1,43 @@
 import './Home.css';
 import { SingleList } from '../components';
-export function Home({ data, setListPath }) {
+import { useState } from 'react';
+import { createList } from '../api';
+import { useNavigate } from 'react-router-dom';
+
+export function Home({ data, setListPath, user }) {
+	const navigate = useNavigate();
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** such a great idea to use optional chaining for error prevention!‏